Tests/Executor/JobExecutorInlineConfigTest.php

```diff
@@ -356,56 +351,56 @@ public function tagOverrideTestDataProvider()
             'storedConfigTag' => null,
             'requestConfigTag' => null,
             'requestParamsTag' => null,
-            'expectedOverrideVersion' => null,
+            'expectedVersion' => '1.4.0',
```

Comment:
Tady me ted trochu strasi ta hodnota `1.4.0`, protoze vubec netusim, kde se vzala.

Reply:
No bohužel no, taky mě to dráždí - ta se vezme z konfigurace komponenty. Dostat tam jinou bude v téhle konstelaci těžký (není tam na to nachystanej mock), nicméně napadla mě v souvislosti s tím taková nepěkná díra, pošlu ještě updat

Reply:
Ta díra je tohle 
![image](https://user-images.githubusercontent.com/4319320/115238196-b358d900-a11d-11eb-8104-c92fabd5437a.png)

Reply:
Asi to nevidim, co je ta dira? Ze se pouzije verze `1.4.0` i kdyz se v konfigu uvadi `1.2.5`? To je to co jsme resili v patek a myslel jsem, ze je to expected.. poslu inline config, tak se stored config uplne ignoruje.

Reply:
Ano presne tak, ale z toho testu to absolutne neni zrejmy.  Proto jsem to tam dal bokem https://github.com/keboola/docker-bundle/pull/563#discussion_r615815408 (a muze se to klidne vyhodit)
Ta "díra" (nepřesné vyjádření) spočívá v tom, že v tomhle testu (Tests/Executor/JobExecutorInlineConfigTest.php) *žádná* hodnota `storedConfigTag` nemá vůbec žádný efekt, takže tam byla půlka těch provided values nesmyslně